data/pid_data.py

```diff
@@ -2498,6 +2566,214 @@
+								  }]},
+       'get_sub_device_range': 0,
+       'name': 'FOLLOW_SPOT_CONTROLLER_MODE',
+       'set_request': {'items': [{'name': 'followspot_control', 'type': 'uint8',
+								 'labels': [(0, 'Normal Operation'),
+											(1, 'Intensity Control'),
+											(2, 'Iris Control'),
+											(3, 'Intensity + Iris Control'),
+											(4, 'Zoom Control'),
+											(5, 'Intensity + Zoom Control'),
+											(6, 'Iris + Zoom Control'),
+											(7, 'Intensity + Iris + Zoom Control'),
+											(8, 'Edge Control'),
+											(9, 'Intensity + Edge Control'),
+											(10, 'Iris + Edge Control'),
+											(11, 'Intensity + Iris + Edge Control'),
+											(12, 'Edge + Zoom Control'),
+											(13, 'Intensity + Edge + Zoom Control'),
+											(14, 'Iris + Edge + Zoom Control'),
+											(15, 'Intensity + Iris + Edge + Zoom Control')],
+								 }]},
+       'set_response': {'items': []},
+       'set_sub_device_range': 0,
+       'link': '',
+       'value': 34312},
+			 
+	  {'get_request': {'items': []},
+       'get_response': {'items': [{'name': 'zoom_type', 'type': 'uint8',
+								  'labels': [(0, 'Fast'),
+											 (1, 'In Focus'),
+								  }]},
+       'get_sub_device_range': 0,
+       'name': 'ZOOM_TYPE',
+       'set_request': {'items': [{'name': 'followspot_control', 'type': 'uint8',
+								 'labels': [(0, 'Fast'),
+											(1, 'In Focus'),
+								 }]},
+       'set_response': {'items': []},
+       'set_sub_device_range': 0,
+       'link': '',
+       'value': 34336},
+			 
+	  {'get_request': {'items': []},
+       'get_response': {'items': [{'name': 'zoom_table', 'type': 'uint8',
+								  'labels': [(0, '30ft Throw'),
+											 (1, '50ft Throw'),
+											 (2, '75ft Throw'),
+											 (3, '100ft Throw'),
+											 (4, '300ft Throw'),
+											 (5, '15ft Throw'),
+								  }]},
+       'get_sub_device_range': 0,
+       'name': 'FOLLOW_SPOT_CONTROLLER_MODE',
+       'set_request': {'items': [{'name': 'zoom_table', 'type': 'uint8',
+								  'labels': [(0, '30ft Throw'),
+											 (1, '50ft Throw'),
+											 (2, '75ft Throw'),
+											 (3, '100ft Throw'),
+											 (4, '300ft Throw'),
+											 (5, '15ft Throw'),
+								 }]},
+       'set_response': {'items': []},
+       'set_sub_device_range': 0,
+       'link': '',
+       'value': 34337},
+			 
+	  {'name': 'ZERO_GOBO_WHEELS',
+       'set_request': {'items': [{'name': 'followspot_control', 'type': 'bool'}]},
+       'set_response': {'items': []},
+       'set_sub_device_range': 0,
+       'link': '',
+       'value': 34368},
+			 
+	  {'name': 'CLEAR_LOG',
+       'set_request': {'items': [{'name': 'clear_log', 'type': 'bool'}]},
+       'set_response': {'items': []},
+       'set_sub_device_range': 0,
+       'link': '',
+       'value': 34400},
+			 
+	  {'get_request': {'items': []},
+       'get_response': {'items': [{'name': 'disable_mechanics', 'type': 'uint8'}]},
```

Comment:
All this info I've got from the fixtures itself, not from documents. For some values I don't have an explanation, but in my opinion it is better to have something then nothing.

Mvg,

Bart van Stiphout

> On 16 mei 2015, at 19:24, Simon Newton <notifications@github.com> wrote:
> 
> In data/pid_data.py:
> 
> > +	  {'name': 'ZERO_GOBO_WHEELS',
> > +       'set_request': {'items': [{'name': 'followspot_control', 'type': 'bool'}]},
> > +       'set_response': {'items': []},
> > +       'set_sub_device_range': 0,
> > +       'link': '',
> > +       'value': 34368},
> > +			 
> > +	  {'name': 'CLEAR_LOG',
> > +       'set_request': {'items': [{'name': 'clear_log', 'type': 'bool'}]},
> > +       'set_response': {'items': []},
> > +       'set_sub_device_range': 0,
> > +       'link': '',
> > +       'value': 34400},
> > +			 
> > +	  {'get_request': {'items': []},
> > +       'get_response': {'items': [{'name': 'disable_mechanics', 'type': 'uint8'}]},
> Should this be a bool ?
> 
> —
> Reply to this email directly or view it on GitHub.
>